airbyte-local.sh

```diff
@@ -200,15 +245,21 @@ function readSrc() {
 }
 
 function checkSrc() {
-    if [ "$check_src_connection" = true ]; then
-        echo "Validating connection to source..."
+    if [[ "$check_src_connection" = true ]]; then
+        log "Validating connection to source..."
         connectionStatusInfo=$(docker run --rm -v "$tempdir:/configs" "$src_docker_image" check --config "/configs/$src_config_filename")
         connectionStatus=$(echo "$connectionStatusInfo" | jq -r '.connectionStatus.status')
-        if [ "$connectionStatus" != 'SUCCEEDED' ]; then
-            echo $(echo "$connectionStatusInfo" | jq -r '.connectionStatus.message')
-            exit 1;
+        if [[ "$connectionStatus" != 'SUCCEEDED' ]]; then
+            err $(echo "$connectionStatusInfo" | jq -r '.connectionStatus.message')
+            fail
         fi
-        echo "Connection validation successful"
+        log "Connection validation successful"
+    fi
+}
+
+function debug() {
+    if [[ "$debug" = true ]]; then
+        log "$*"
```

Comment:
Should we handle this like the other log levels? We could give it its own color and have it print `DEBUG` instead of info.
```sh
function debug() {
    if ((debug)); then
        fmtLog "debug"
        printLog "$*"
    fi
}
```